deploy-board/deploy_board/templates/configs/new_capacity_adv.html

```diff
@@ -135,9 +135,8 @@ <h4 class="panel-title pull-left">Capacity</h4>
         hostTypeOptions: info.hostTypes.map(
             function (item, idx) {
                 return {
-                    value: item.id,
-                    text: item.abstract_name + " (" + item.core + " cores, " + item.mem + " GB, " + item.storage + ", " + item.provider + ": " + item.provider_name + ")",
-                    isSelected: item.abstract_name === info.defaultHostType
+                    value: item.id, text: item.abstract_name + " (" + item.core + " cores, " + item.mem + " GB, " + item.storage + ", " + item.provider + ": " + item.provider_name + ")",
+                    isSelected: item.abstract_name === capacityCreationInfo.hostTypes.find(hostType => hostType.abstract_name === capacityCreationInfo.defaultHostType).id
```

Comment:
comparing abstract_name with id ?

Reply:
Just checking if that was used as a side effect somewhere, not using that